src/app/auth/authentication.service.ts

```diff
@@ -100,7 +100,7 @@ export class AuthenticationService {
       let refreshInMs = Math.round(refreshInSeconds * .9) * 1000;
       console.log('Refreshing token in: ' + refreshInMs + ' milliseconds.');
       this.refreshInterval = refreshInMs;
-      if (process.env.ENV !== 'inmemory') {
+      if (process.env.ENV !== 'inmemory' && process.env.ENV !== 'FABRIC8_PLANNER_TEST') {
```

Comment:
Since the login client is generic it shouldn't need know the internals of its users (planner, analysis etc .. ). Could we rely on a generic flag to suppress the refreshToken()? How about something like ... 

```typescript
if ( ! (process.env.NODE_ENV === 'test' && process.env.LOGIN_CLIENT_NO_REFRESH === "true")) {
   // ...  refresh token
}
```

Reply:
@sthaha Thanks! Using `LOGIN_CLIENT_NO_REFRESH` would make more sense.

Reply:
@sthaha If we use `NODE_ENV` and `LOGIN_CLIENT_NO_REFRESH` the developer would have to set 2 environment variables to do one thing. Can't we use only `LOGIN_CLIENT_NO_REFRESH`? 

Also `NODE_ENV='test'` is being used by webpack.test.js[0]. I'm not sure what might be the result of setting `NODE_ENV=test`

[0] - https://github.com/fabric8-ui/ngx-login-client/blob/master/config/webpack.test.js#L21

Reply:
> Also NODE_ENV='test' is being used by webpack.test.js[0]. I'm not sure what might be the result of setting NODE_ENV=test 

Well the idea is to allow user to disable  token refresh **only** when we are running `test`s (i.e. NODE_ENV = test ) and not when the env is production or any other NODE_ENV, perhaps this can be enabled for dev as well. 

BTW isn't the real fix to run token refresh [outside angular zone](http://www.protractortest.org/#/timeouts#angular) as told in the protractor documentation?  Have you considered implementing that?

Reply:
I'm still not sure about using `NODE_ENV == test`.

About the angular zone, I tried but didn't have any luck. Using environment variable is the simplest fix I can think of.